cmd/chartsvc/handler.go

```diff
@@ -95,39 +95,39 @@ func listRepoCharts(w http.ResponseWriter, req *http.Request, params Params) {
 func getChart(w http.ResponseWriter, req *http.Request, params Params) {
 	db, closer := dbSession.DB()
 	defer closer()
-	var chart *models.Chart
```

Comment:
what's the reasoning behind these pointer to value changes?

Reply:
I was having trouble getting the mock interface match the expectation description. Simplest was to fix this was to remove the double reference.

```console
mock: Unexpected Method Call
-----------------------------

One(**models.Chart)
		0: (**models.Chart)(0xc4200b41a8)

The closest call I have is: 

One(*models.Chart)
		0: &models.Chart{ID:"", Name:"", Repo:models.Repo{Name:"", URL:""}, Description:"", Home:"", Keywords:[]string(nil), Maintainers:[]models.Maintainer(nil), Sources:[]string(nil), Icon:"", RawIcon:[]uint8(nil), ChartVersions:[]models.ChartVersion(nil)}
```

Reply:
hmm okay, I think the tests in monocular had a similar thing though, but it's not a big problem either way.